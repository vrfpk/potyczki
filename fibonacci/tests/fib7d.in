919463694325529
