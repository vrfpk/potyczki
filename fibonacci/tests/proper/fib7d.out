2057056224968171
